main.py

```diff
@@ -1,12 +1,11 @@
-﻿from asyncio.windows_events import NULL
-from cmath import inf
-
+﻿from cmath import inf
 
 def getOpenCells(board):  # Required for unit-tests
-    open_cells = 0    
-    open_cells += len([1 for i in range(len(board)) for j in range(len(board)) if board[i][j] == ' '])
```

Comment:
Consider storing the board via numbers instead of chars - that will decrease the number of `if board[i][j] == 'X':` etc.